signalfx/resource_signalfx_dashboard_group.go

```diff
@@ -592,14 +598,16 @@ func dashboardgroupUpdate(d *schema.ResourceData, meta interface{}) error {
 	log.Printf("[DEBUG] SignalFx: Update Dashboard Group Response: %v", dg)
 
 	d.SetId(dg.Id)
-	return dashboardGroupAPIToTF(d, dg)
+	return dashboardGroupAPIToTF(d, dg, meta)
 }
 
 func getNonMirroredDashes(config *signalfxConfig, d *schema.ResourceData) ([]*dashboard_group.DashboardConfig, error) {
-	var mirrorIDsToBeOmitted map[string]bool
-	if d.HasChange("dashboard") {
-		oldDashboardList, newDashboardList := d.GetChange("dashboard")
-		mirrorIDsToBeOmitted = getMirrorsToBeOmitted(oldDashboardList, newDashboardList)
+	mirrorIDsToBeOmitted := map[string]bool{}
+	mirroredDashboardConfigs, err := getMirroredDashboardConfigs(config, d)
```

Comment:
Is it safe to ignore errors returned by this method? It appears we only handle the happy path here. Would it make sense to log those errors?

Reply:
I thought it is okay to proceed with assuming all dashboards are non-mirror if we cannot fetch enough information. Now I think about it again, it should fail instead of proceeding because TF wii be tracking wrong info if we do so. I'll change it.